### ISSUE TITLE
credits need to be filled in

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/least-squares-regression#16]
Credits need to be filled in, see simOptions.credits in least-squares-regression-main.js.

**Comment by veillette** [comment]
The credits have been filled from the design document.

**Comment by veillette** [comment]
Assigning to @pixelzoom for review

**Comment by pixelzoom** [comment]
Looks good. I made a couple of changes to make the credits follow PhET conventions:

(1) The credits should be listed in alphabetical order. So I reordered 'team'.

(2) The leadDesigner does not need to be included in other categories. They are by default part of the team and QA.  So I removed AM from 'qualityAssurance'.

Assigning back to @bereaphysics in case you have any questions. Feel free to close.
